### DISCUSSION TITLE
Implement Google Analytics (GA4)

### DISCUSSION THREAD

**Discussion by hayleyw7** [nditcommunity/ndit-website#55] [Technology]

**Comment by royemosby** [comment]
[GA4 Overview](https://support.google.com/analytics/answer/10089681?hl=en) for general reference

**Comment by marcusolsson** [comment]
I strongly advise against using Google Analytics. I'd suggest looking into more GDPR-friendly and ethical alternatives like [Fathom](https://usefathom.com/), [Plausible](https://plausible.io/), or [Simple Analytics](https://www.simpleanalytics.com/).

The downside is that these don't have free tiers.

**Reply by hayleyw7** [reply]
I only have experience with GA, but if someone wants to take charge on implementing another tool, I have no concerns with that, as long as it's free. :)

Edit: Just to clarify though, I did since mention a bit below about being aware of gatekeeping in terms of selecting niche tools.

**Comment by royemosby** [comment]
There also [Piwik](https://piwik.pro/) which I bring up because it has a free core service that may suffice. It also mentions [privacy compliance](https://piwik.pro/privacy-compliance/) on its brochure site.

**Reply by hayleyw7** [reply]
Have you used it? We do probably want to be aware of accidentally gatekeeping contributors out of the project due to the choice of many niche tools and techs.

I lean toward using the most common free tool for every purpose, with the most online help and most valuable resume bullet to add for junior contributors.

**Comment by royemosby** [comment]
@hayleyw7 are there any specific metrics that you are trying to hone in on? I don't have experience in this space other than tracking generic hits.

**Reply by hayleyw7** [reply]
Just what people do on the site once they're there. Do they just click join? Do they click a tab? I was thinking of having an event name for each user click event.
